django_date_extensions/fields.py

```diff
@@ -172,20 +221,27 @@ def clean(self, value):
         if isinstance(value, ApproximateDate):
             return value
         value = re.sub('(?<=\d)(st|nd|rd|th)', '', value.strip())
-        for format in settings.DATE_INPUT_FORMATS:
+        for date_format in settings.DATE_INPUT_FORMATS:
```

Comment:
Thanks, this is on master in its own commit at 0d2d4be5d2224807c3e68cc2e740752416f66ea0